test/Microsoft.OpenApi.Readers.Tests/V2Tests/OpenApiDocumentTests.cs

```diff
@@ -238,5 +239,30 @@ public void ShouldParseProducesInAnyOrder()
                 }
             }
         }
+
```

Comment:
Can we add one more test case with the mixture of global produces and operation-level produces?

Reply:
This is already covered by twoResponses test

Reply:
I see. 

Looks like the twoResponses test has only `application/json` as the produce type in all places. It would be great to have `application/xml` in one of the operation-level produce to test that there are no issues with such interaction. There's no test that covers such case right now.

Any chance you could help fix the test? It should not be too big of a change.

---
In reply to: [275497172](https://github.com/Microsoft/OpenAPI.NET/pull/398#discussion_r275497172) [](ancestors = 275497172)

Reply:
Sure